cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaMirrorMaker2Connectors.java

```diff
@@ -326,4 +330,18 @@ private static String addTLSConfigToMirrorMaker2ConnectorConfig(Map<String, Obje
             return "PLAINTEXT";
         }
     }
+
+    private static boolean hasMatchingBootstrapServers(KafkaMirrorMaker2 kafkaMirrorMaker2, String connectClusterAlias, String targetClusterAlias) {
+        List<String> configs = Optional.ofNullable(kafkaMirrorMaker2)
```

Comment:
Why do we need `Optional.ofNullable`? I think `kafkaMirrorMaker2 cannot be null at this point.